test/util/long.test.ts

```diff
@@ -15,8 +15,7 @@ import {
   multiplyDivideCeil,
   multiplyDivideRound,
   multiplyDivide
-} from '../src/util/long'
-require('source-map-support').install()
```

Comment:
I often wondered about these `source-map-support` install()s littered throughout. 
It's better to leave to the library client, no?
It looks like you've removed them all